### PR TITLE
ISPN-9893 Reactive API second round

### PR DESCRIPTION
**Pull request by karesti**
- Remove reactive from the packaging

### CODE REVIEW

File: api/src/main/java/org/infinispan/api/store/KeyValueStore.java [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-import org.infinispan.api.reactive.listener.KeyValueStoreListener;
-import org.infinispan.api.reactive.query.QueryRequest;
+import org.infinispan.api.store.listener.KeyValueStoreListener;
+import org.infinispan.api.store.query.QueryRequest;
 import org.reactivestreams.Publisher;
 
 /**
  * A Reactive Key Value Store provides a highly concurrent and distributed data structure, non blocking and using
- * reactive streams.
+ * store streams.
```
Comment (pruivo) [inline api/src/main/java/org/infinispan/api/store/KeyValueStore.java]: what is `store streams`?

Comment (pruivo) [reply api/src/main/java/org/infinispan/api/store/KeyValueStore.java]: ps. there is `store streams` everywhere now

File: core/src/test/java/org/infinispan/reactive/publisher/impl/RehashClusterPublisherManagerTest.java [hunk 2]
```diff
@@ -43,7 +43,7 @@
  * @author wburns
  * @since 10.0
  */
-@Test(groups = "functional", testName = "reactive.publisher.impl.RehashClusterPublisherManagerTest")
+@Test(groups = "functional", testName = "store.publisher.impl.RehashClusterPublisherManagerTest")
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/reactive/publisher/impl/RehashClusterPublisherManagerTest.java]: why the change in the test name?

File: feature-pack/embedded/src/main/resources/modules/org/infinispan/core/slot/module.xml [hunk 3]
```diff
@@ -12,7 +12,7 @@
         <module name="org.apache.xerces" services="import"/>
         <module name="org.infinispan.commons" slot="@infinispan.module.slot@" />
         <module name="com.github.ben-manes.caffeine" slot="@infinispan.module.slot@"/>
-        <!--Export rxjava and reactive streams for store implementations-->
+        <!--Export rxjava and store streams for store implementations-->
```
Comment (pruivo) [inline feature-pack/embedded/src/main/resources/modules/org/infinispan/core/slot/module.xml]: I'm pretty sure it should be `reactive streams` here :)

File: integrationtests/security-it/src/test/java/org/infinispan/test/integration/security/utils/Deployments.java [hunk 4]
```diff
@@ -44,7 +44,7 @@ public static WebArchive createBaseTestDeployment() {
                   new File("target/test-libs/infinispan-commons.jar"),
                   new File("target/test-libs/caffeine.jar"),
                   new File("target/test-libs/rxjava.jar"),
-                  new File("target/test-libs/reactive-streams.jar"),
+                  new File("target/test-libs/store-streams.jar"),
```
Comment (pruivo) [inline integrationtests/security-it/src/test/java/org/infinispan/test/integration/security/utils/Deployments.java]: this should be `reactive-streams.jar`

File: feature-pack/embedded/src/main/resources/modules/org/infinispan/core/slot/module.xml (hunk 3 above)
Comment (karesti) [reply feature-pack/embedded/src/main/resources/modules/org/infinispan/core/slot/module.xml]: thnaks to intellij
I will redo things here anyway

**Comment by pruivo** [comment]
@karesti needs rebase...

**Comment by karesti** [comment]
waiting to rework
